src/qcommon/cm_load.c

```diff
@@ -456,15 +456,51 @@ void CMod_LoadBrushSides(lump_t *l)
 	}
 }
 
+/**
+ * @brief CMod_LoadCustomEntityString
+ * @param[in] name
+ */
+qboolean CMod_LoadCustomEntityString(const char* name) {
+	fileHandle_t file;
+	int entFileLen = 0;
+	char* filename;
+	char noext[MAX_QPATH];
+
+	COM_StripExtension(name, noext);
+	filename = va("%s.ent", noext);
+
+	entFileLen = FS_FOpenFileRead(filename, &file, qtrue);
+
+	if (file && entFileLen > 0)
+	{
+		cm.entityString = (char*)Hunk_Alloc(entFileLen + 1, h_high);
+		cm.numEntityChars = entFileLen + 1;
```

Comment:
`numEntityChars` is commented out in `clipMap_t` struct in ETL, this does not compile without uncommenting that.

https://github.com/etlegacy/etlegacy/blob/f1e9d75914cb983b57d7f98c04f11ce860f9fb99/src/qcommon/cm_local.h#L166